### PR TITLE
resolved test widget and added const keyword

### PR DESCRIPTION
**Pull request by buzbunny**

### CODE REVIEW

File: frontend/lib/WelcomeScreen.dart
Comment (P-Asare) [inline frontend/lib/WelcomeScreen.dart:1]: Realised there were scroll issues when the keyboard popped up. 
I'll take a look at them though. Good job. 💪

**Review by P-Asare — COMMENTED** [review]
Looks good to me. 💪

File: frontend/lib/WelcomeScreen.dart
Comment (P-Asare) [reply frontend/lib/WelcomeScreen.dart:1]: @buzbunny  I went ahead and made the necessary changes to allow for scrolling. Do well to confirm that it's fine.

**Review by P-Asare — APPROVED** [review]
I think it looks even better now. 💪

**Review by Joseph-Lartey — APPROVED** [review]

**Review by kwakuNti — APPROVED** [review]
looks good

**Pull request merged**